.github/workflows/changes.yml

```diff
@@ -104,6 +104,8 @@ on:
         value: ${{ jobs.int_tests.outputs.nginx }}
       opentelemetry:
         value: ${{ jobs.int_tests.outputs.opentelemetry }}
+      openobserve:
```

Comment:
`int_tests` is a GitHub job that detects if a branch contains changes to integration tests. Since you didn't added any integration tests dedicated to `openobserve` you should  revert all changes to `.github/workflows/*`,

Reply:
make sense